Assignment7/Assignment7_QNo1/SolAssignmentNo7_QNo1.rb

```diff
@@ -0,0 +1,23 @@
+# => 1. Define a lambda which will take 2 arguments
+# => first name and last name and will generate fullname.
+class ClassDemo
+  def lambda_operation
+    # => define lambda
+    full_name = -> (fname, lname) { puts "#{fname} #{lname}" }
```

Comment:
@HrisikeshSinghaChowdhury puts is for only displaying the value, it will not return the value. to return the value from the method you should use the value as the last statement of that method
eg: full_name = -> (fname, lname) { "#{fname} #{lname}" }

Reply:
done